client_test/virtualfund_test.go

```diff
@@ -47,11 +47,66 @@ func TestVirtualFundIntegration(t *testing.T) {
 	chainServiceA := chainservice.NewMockChainService(chain, alice.Address())
 	chainServiceB := chainservice.NewMockChainService(chain, bob.Address())
 	chainServiceI := chainservice.NewMockChainService(chain, irene.Address())
+	chainServiceBr := chainservice.NewMockChainService(chain, brian.Address())
 	broker := messageservice.NewBroker()
 
 	clientA, _ := setupClient(alice.PrivateKey, chainServiceA, broker, logDestination, 0)
+	irene, _ := setupClient(irene.PrivateKey, chainServiceI, broker, logDestination, 0)
+	ivan, _ := setupClient(brian.PrivateKey, chainServiceBr, broker, logDestination, 0)
 	clientB, _ := setupClient(bob.PrivateKey, chainServiceB, broker, logDestination, 0)
-	clientI, _ := setupClient(irene.PrivateKey, chainServiceI, broker, logDestination, 0)
 
-	openVirtualChannels(t, clientA, clientB, clientI, 1)
+	openN_HopVirtualChannels(t, []client.Client{clientA, irene, ivan, clientB}, 1)
+}
+
+// openN_HopVirtualChannels connects the n given participants in a line of ledger channels,
+// then uses these ledger connections to open channels between the first participant
+// and each other participant.
+//
+// This makes channels with 0, 1, ..., [len(participants) - 1] hops.
+func openN_HopVirtualChannels(t *testing.T, participants []client.Client, channelsPerCounterparty uint) {
```

Comment:
The `channelsperCounterparty` is not documented, and it is also not being used nontrivially here as far as I can tell? Maybe we should drop it until we need it?

Reply:
I have renamed this to `concurrency` in some follow-up work, which is in line with usage on the testground repo.

Moving forward, I'd like to set up a simple script that shoots out graphs like the ones I had put in slack yesterday:

![image](https://user-images.githubusercontent.com/10780590/195907353-7d5561c8-f9b9-4073-8339-fd4345f1f785.png)